Release 2 0 11/merge branch v1 to main

maheshrajamani opened the pull request:
**What this PR does**:
Merging v1 to v2

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

tatu-at-datastax commented on CHANGELOG_V1.md:
```diff
@@ -1,5 +1,34 @@
+
+- Use concurrency to cancel workflows on pull request updates [\#2493](https://github.com/stargate/stargate/issues/2493)
+- Update to DSE 6.8.33 [\#2490](https://github.com/stargate/stargate/issues/2490)
+- Update to DSE 6.8.33 [\#2487](https://github.com/stargate/stargate/issues/2487)
```
Probably want to remove 2 of these?

tatu-at-datastax replied:
Oh, if these are already in V1 release notes never mind.

maheshrajamani replied:
Aren't these should be handled in v1 branch? else we will have conflict every time.

tatu-at-datastax replied:
Yes, correct.

ivansenic commented:
@maheshrajamani I don't understand how these changes are not picked up with the merge: https://github.com/stargate/stargate/pull/2513/files

When I tried to do a merge this was picked up fine, what command did you use and how do you resolve conflicts?

Now the state of that file is not same on `v1` and `main`.. We need to resolve this, because in future more important changes could be simply ignored and this must not happen.

maheshrajamani commented:
@ivansenic Below are the commands I ran. I did select all and did `Accept Yours` for resolving merge conflict since only pom files showed for conflict.

`git checkout -b release-2-0-11/merge-branch-v1-to-main
Switched to a new branch 'release-2-0-11/merge-branch-v1-to-main'`
`stargate % git merge origin/v1
CONFLICT (modify/delete): coordinator/testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java deleted in HEAD and modified in origin/v1. Version origin/v1 of coordinator/testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java left in tree.
Auto-merging coordinator/testing/pom.xml
CONFLICT (content): Merge conflict in coordinator/testing/pom.xml
Auto-merging coordinator/testing-services/pom.xml
CONFLICT (content): Merge conflict in coordinator/testing-services/pom.xml
Auto-merging coordinator/stargate-starter/pom.xml
CONFLICT (content): Merge conflict in coordinator/stargate-starter/pom.xml
Auto-merging coordinator/restapi/pom.xml
...
...`
